addons/backgrounds/src/containers/BackgroundSelector.tsx

```diff
@@ -78,7 +78,14 @@ export const BackgroundSelector: FunctionComponent = memo(() => {
   );
 
   const [globals, updateGlobals] = useGlobals();
-
+  if (!globals[BACKGROUNDS_PARAM_KEY] && backgroundsConfig.values.length) {
+    const defaultSetting = backgroundsConfig.values.find(
+      (value) => value.name === backgroundsConfig.default
+    );
+    globals[BACKGROUNDS_PARAM_KEY] = {
+      value: defaultSetting.value,
+    };
```

Comment:
This isn't right, we should not be mutating globals

Reply:
Is calling `updateGlobals` a good solution?

Reply:
No we should not put the background value coming from parameters into the global.

What we want to do instead is read both in the place where we apply the background:

If the global.background is set, apply the CSS value to that.
if the global.background is null, but the parameter is set, apply the CSS value to that.
if neither have a valid value, clear the CSS styles applied, so there is no background color applied at all.